test/unit/tracks/text-track-display.test.js

```diff
@@ -0,0 +1,279 @@
+/* eslint-env qunit */
+import Html5 from '../../../src/js/tech/html5.js';
+import Component from '../../../src/js/component.js';
+
+import * as browser from '../../../src/js/utils/browser.js';
+import TestHelpers from '../test-helpers.js';
+import document from 'global/document';
+import sinon from 'sinon';
+
+QUnit.module('Text Track Display', {
+  beforeEach(assert) {
+    this.clock = sinon.useFakeTimers();
+  },
+  afterEach(assert) {
+    this.clock.restore();
+  }
+});
+
+const getTrackByLanguage = function(player, language) {
+  const tracks = player.tech_.remoteTextTracks();
+
+  for (let i = 0; i < tracks.length; i++) {
+    const track = tracks[i];
+
+    if (track.language === language) {
+      return track;
+    }
+  }
+};
+
+const getMenuItemByLanguage = function(items, language) {
+  for (let i = items.length - 1; i > 0; i--) {
+    const captionMenuItem = items[i];
+    const trackLanguage = captionMenuItem.track.language;
+
+    if (trackLanguage && trackLanguage === language) {
+      return captionMenuItem;
+    }
+  }
+};
+
+QUnit.test('if native text tracks are not supported, create a texttrackdisplay', function(assert) {
+  const oldTestVid = Html5.TEST_VID;
+  const oldIsFirefox = browser.IS_FIREFOX;
+  const oldTextTrackDisplay = Component.getComponent('TextTrackDisplay');
+  const tag = document.createElement('video');
+  const track1 = document.createElement('track');
+  const track2 = document.createElement('track');
+
+  track1.kind = 'captions';
+  track1.label = 'en';
+  track1.language = 'English';
+  track1.src = 'en.vtt';
+  tag.appendChild(track1);
+
+  track2.kind = 'captions';
+  track2.label = 'es';
+  track2.language = 'Spanish';
+  track2.src = 'es.vtt';
+  tag.appendChild(track2);
+
+  Html5.TEST_VID = {
+    textTracks: []
+  };
+
+  browser.IS_FIREFOX = true;
+
+  const fakeTTDSpy = sinon.spy();
+
+  class FakeTTD extends Component {
+    constructor() {
+      super();
+      fakeTTDSpy();
+    }
+  }
+
+  Component.registerComponent('TextTrackDisplay', FakeTTD);
+
+  const player = TestHelpers.makePlayer({}, tag);
+
+  assert.strictEqual(fakeTTDSpy.callCount, 1, 'text track display was created');
+
+  Html5.TEST_VID = oldTestVid;
+  browser.IS_FIREFOX = oldIsFirefox;
+  Component.registerComponent('TextTrackDisplay', oldTextTrackDisplay);
+
+  player.dispose();
+});
+
+QUnit.test('no captions tracks, no captions are displayed', function(assert) {
```

Comment:
You're right! Turns out there's a test like this in `text-track-controls.test.js` already which I will expand to check if `remoteTextTracks` is empty as well.